main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/labstack/echo"
+
+	"github.com/AlejandroSeguraWIZ/academy-go-q12021/registry"
+	"github.com/AlejandroSeguraWIZ/academy-go-q12021/router"
+)
+
+func main() {
+	address := ":8081"
```

Comment:
I'd recommend having a configuration file for values like these. After creating it, you can use [viper](https://github.com/spf13/viper) to make use of it.